CONTRIBUTING.md

```diff
@@ -58,9 +58,37 @@ This project is built from several repositories:
 * [percona/pmm-qa](https://github.com/percona/pmm-qa)
 * [percona/pmm-doc](https://github.com/percona/pmm-doc)
 
-## API documentation
+## API Reference Documentation
 
-See API definitions [here](https://percona-pmm.readme.io/reference/introduction).
+You can review the PMM API definition [here](https://percona-pmm.readme.io/).
+
+The API documentation source files are located in the `markdown` directory. The
```

Comment:
I think we also should mention that API documentation is generated from the swagger.json file.

Reply:
Good idea, I'll add that.

Reply:
Done.